docs/quick-start.md

```diff
@@ -0,0 +1,197 @@
+3. Copy the sample inventory hosts file: `cp examples/hosts inventory/hosts`
+4. Edit the configuration file (`inventory/host_vars/matrix.example.com/vars.yml`)
+5. Edit the inventory hosts file (`inventory/hosts`)
+
+Before editing those two files, make sure to read explanations on them to understand what needs to be configured.
+
+**💡Recommendation**: instead of configuring a lot of things all at once, we recommend starting with the basic (default) settings in order to get yourself familiar with how the playbook works. After making sure that everything works as expected, you can add (and remove) advanced settings / features and run the playbook as many times as you wish.
+
+**Notes:**
+- If you are not in control of anything on the base domain, you would need to set additional configuration on `vars.yml`. For more information, see [How do I install on matrix.example.com without involving the base domain?](faq.md#how-do-i-install-on-matrix-example-com-without-involving-the-base-domain) on our FAQ.
+- When you edit `vars.yml`, make sure to pick a correct domain, because you can't change the domain after deployment. If you pick the wrong one, you'll have to run the Uninstalling step.
+
+## Install
+
+<sup>This section is optimized for this quick-start guide and is derived from the following full-documentation page: [Installing](installing.md)</sup>
+
+After editing `vars.yml` and `hosts` files, let's start the **installation** procedure.
+
+### Update Ansible roles
+
+Before installing, you need to update the Ansible roles in this playbook. To update them, run `just roles` on your local computer.
+
+### Run installation command
+
+Then, run the command below to start installation:
+
+````sh
+ansible-playbook -i inventory/hosts setup.yml --tags=install-all,ensure-matrix-users-created,start
+````
+
+If you **don't** use SSH keys for authentication, but rather a regular password, you may need to add `--ask-pass` to the command.
+
+If you **do** use SSH keys for authentication, **and** use a non-root user to *become* root (sudo), you may need to add `-K` (`--ask-become-pass`) to the command.
+
+Wait until the command completes. If it's all green, everything should be running properly.
+
+## Create your user account
+
+<sup>This section is optimized for this quick-start guide and is derived from the following full-documentation page: [Registering users](registering-users.md)</sup>
+
+As you have configured your brand new server and the client, you need to **create your user account** on your Matrix server.
+
+To register a user via this Ansible playbook, run the command below on your local computer.
+
+**Notes**:
+- Make sure to adjust `YOUR_USERNAME_HERE` and `YOUR_PASSWORD_HERE`
+- For `YOUR_USERNAME_HERE`, use a plain username like `john`, not your full identifier (`@user:example.com`)
+- Use `admin=yes` or `admin=no` depending on whether you wish to make the user an administrator of the Matrix server
```

Comment:
I have two questions here:

- Is it clear which effect making a user an admin of the Matrix server has? (I guess you should never make a user admin casually)
- Should you make yourself (= the first user of the Matrix server) an administrator? Is it rather optional?

Reply:
This section is named "Create your user account" which may not be great, because you may wish to create multiple accounts here. While it's possible, it doesn't seem obvious.

Yes, you will most likely wish to create your first user account as an administrator. The person who sets up the Matrix server will definitely wish to create their account as an administrator.

You're right that we don't explain what the effect is and that one shouldn't make users administrators willy-nilly.

Reply:
> This section is named "Create your user account" which may not be great, because you may wish to create multiple accounts here. While it's possible, it doesn't seem obvious.

I would add: `For an instruction to add multiple user accounts, see [Registering users](docs/registering-users.md)`, because creating other accounts than your user account is out of the scope of this document, whose objects are listed at the top.